src/QueryType/Select/Query/FilterQuery.php

```diff
@@ -205,7 +205,7 @@ protected function init()
     {
         foreach ($this->options as $name => $value) {
             switch ($name) {
-                case 'key':
```

Comment:
We explicitly didn't deprecate the 'key' on FilterQuery:

```php
                case 'key':
                    break;
                    if ($this instanceof FilterQuery) {
                        break;
                    }
                    @trigger_error('setting local parameter using the "key" option is deprecated in Solarium 5 and will be removed in Solarium 6. Use "local_key" instead', E_USER_DEPRECATED);
```

Reply:
yes, you're right, missed that one.
reverted...